### PR TITLE
Use request.session for all requests, connection has to be made with 'with' statement

### PR DESCRIPTION
**Pull request by npklein**
to make sure that the password file gets cleaned up after use

### CODE REVIEW

File: molgenis-python/src/main/resources/python/security.py [hunk 1]
```diff
@@ -1,29 +1,35 @@
 from getpass import getpass
 from pbkdf2 import PBKDF2
-import crypto
-import sys
-sys.modules['Crypto'] = crypto
-from Crypto.Cipher import AES
 import os
+import platform
+if platform.system() == 'Darwin':
```
Comment (fdlk) [inline molgenis-python/src/main/resources/python/security.py]: Bit specific, this

**Comment by fdlk** [comment]
Can one of the admins verify this patch?

**Comment by fdlk** [comment]
add to whitelist

**Comment by dennishendriksen** [comment]
Could you merge with master? Lets have a chat about when to merge your stuff.

**Pull request merged**